### ISSUE TITLE
Create API document detailing what the library provides

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-py#1]
Should this library provide the same functionality as the [metadig-r](https://github.com/NCEAS/metadig-r) library? If yes then maybe these two libraries should
be supporting the same set of functions, that would be enumerated in an API document. 

Start by reviewing / implementing the check that have already been implemented in the
metadig-r library.

**Comment by amoeba** [comment]
Good idea. I wrote up a mini-readme of what I've got going so far. It's not everything the package does but it's the set of highest-level, user-facing functions.

https://github.com/NCEAS/metadig-r
